### PR TITLE
Board: Card Attachments 

### PR DESCRIPTION
**Pull request by annano**

### CODE REVIEW

File: plugins/attachment-resources/src/components/AttachmentDroppable.svelte [hunk 1]
```diff
@@ -0,0 +1,67 @@
+<!--
+// Copyright © 2020, 2021 Anticrm Platform Contributors.
+// Copyright © 2021 Hardcore Engineering Inc.
```
Comment (haiodo) [inline plugins/attachment-resources/src/components/AttachmentDroppable.svelte]: Only require this one.
`Copyright © 2022 Hardcore Engineering Inc`

File: plugins/attachment-resources/src/utils.ts [hunk 2]
```diff
@@ -52,7 +58,7 @@ export async function uploadFile (file: File, opts?: { space: Ref<Space>, attach
   return await resp.text()
 }
 
-export async function deleteFile (id: string): Promise<void> {
+export async function deleteFile(id: string): Promise<void> {
```
Comment (haiodo) [inline plugins/attachment-resources/src/utils.ts:61]: indentation

File: plugins/attachment-resources/src/utils.ts [hunk 3]
```diff
@@ -68,7 +74,33 @@ export async function deleteFile (id: string): Promise<void> {
...
+  list: FileList,
+  attachTo: { objectClass: Ref<Class<Doc>>; space: Ref<Space>; objectId: Ref<Doc> }
+) {
+  const { objectClass, objectId, space } = attachTo
+  try {
+    for (let index = 0; index < list.length; index++) {
+      const file = list.item(index)
+      if (file !== null) {
+        const uuid = await uploadFile(file, { space, attachedTo: objectId })
+        console.log('uploaded file uuid', uuid)
```
Comment (haiodo) [inline plugins/attachment-resources/src/utils.ts:88]: Logging not required

**Review by haiodo — CHANGES_REQUESTED** [review]

File: plugins/attachment-resources/src/components/AttachmentDroppable.svelte (hunk 1 above)
Comment (annano) [reply plugins/attachment-resources/src/components/AttachmentDroppable.svelte]: Removed

File: plugins/attachment-resources/src/utils.ts (hunk 2 above)
Comment (annano) [reply plugins/attachment-resources/src/utils.ts:61]: This is how `rushx format` formatting it

File: plugins/attachment-resources/src/utils.ts (hunk 3 above)
Comment (annano) [reply plugins/attachment-resources/src/utils.ts:88]: Removed

**Review by haiodo — APPROVED** [review]

**Pull request merged**